fe/fe-core/src/main/java/com/starrocks/journal/bdbje/BDBJEJournal.java

```diff
@@ -56,6 +56,7 @@ public class BDBJEJournal implements Journal {
     private BDBEnvironment bdbEnvironment = null;
     private CloseSafeDatabase currentJournalDB;
     protected Transaction currentTrasaction = null;
+    private String prefix = ""; // used to distinguish different module's db in BDB
```

Comment:
I think this should be final and not be changed, so it should be set in constructor.

Reply:
done